src/classes/GS_NonprofitTrialOrgService.cls

```diff
@@ -138,10 +138,11 @@ public with sharing class GS_NonprofitTrialOrgService {
      * {"Sandbox":false,"TrialExpiration":"2020-11-29T00:00:00.000Z", "applicationDate":"2020-10-20"} Simulates a trial org which has applied for licenses.
      */
     private static TestingConfig getTestingConfiguration() {
-        List<StaticResource> staticResources = [SELECT Body
-                                                FROM StaticResource
-                                                WHERE Name = :STATIC_RESOURCE_NAME
-                                               ];
+        List<StaticResource> staticResources = [
+            SELECT Body FROM StaticResource
```

Comment:
This clause should only be used on standard and custom objects, not system objects.